### DISCUSSION TITLE
Service account for long running tasks

### DISCUSSION THREAD

**Discussion by DeanKamali** [directus/directus#11380] [Archive]
Hey everyone, 

How can I go about creating a service account with a token that doesn't expire?
it would make it easier to use that account with a third-party script / AWS Lambda to perform a task and later write results back to directus without frequent/unnecessary authentication. 

Thanks

**Comment by haniefss** [comment]
You can create new (service) account, set required role/permissions and use token on edit profile page
![Annotation 2022-02-02 231407](https://user-images.githubusercontent.com/39113199/152192930-1ad9a85d-b9d6-48c4-b256-978b7f9b4bb2.png)
